web/controllers/screeningController/index.ts

```diff
@@ -301,3 +303,106 @@ export async function updateScreenerByMailHandler(
         next();
     }
 }
+
+/**
+ * @api {POST} /courses getCourses
+ * @apiVersion 1.0.1
+ * @apiDescription
+ * 
+ * Retrieves the first 20 courses that match the specified filters.
+ * 
+ * 
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName getCourses
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X POST -H "Token: <AUTHTOKEN>" https://dashboard.corona-school.de/api/screening/courses
+ *
+ * @apiParam (JSON Body) {string=} courseState the course state ("created", "submitted", "allowed", "denied", "cancelled")
```

Comment:
What is the `=` for?  
Also: The docs won't compile because of that

Reply:
It's supposed to be an optional parameter (https://jsdoc.app/tags-param.html) should've checked if it works

Reply:
I just typed it as string|undefined, that should convey the same meaning